### PR TITLE
Re-add locking/visibility tools to edit.js

### PR DESCRIPTION
**Pull request by imgntn**
This PR fixes an accidental regression -- the appropriate locked and visible icons should now appear when appropriate while using edit.js

To test:
1. Open edit.js
2. In the entities list, use the icons on the left to lock some entities and make some invisible.  Try toggling.
3.  Observe that the icons on the table appear correctly.

### CODE REVIEW

File: scripts/system/edit.js [hunk 1]
```diff
@@ -180,18 +179,15 @@ var toolBar = (function() {
         newParticleButton
 
     function initialize() {
-        toolBar = new ToolBar(0, 0, ToolBar.HORIZONTAL, "highfidelity.edit.toolbar", function(windowDimensions, toolbar) {
+        toolBar = new ToolBar(0, 0, ToolBar.VERTICAL, "highfidelity.edit.toolbar", function(windowDimensions, toolbar) {
```
Comment (ZappoMan) [inline scripts/system/edit.js]: hmmm -- I don't think you want that...

**Comment by hifi-gustavo** [comment]
Build console output for commit: 795d866da9555f8151b07705e85c5874de07860c available at: https://gist.github.com/21e8481a17ff1804de42bbd363b48ced<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7785.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7785.dmg

**Comment by imgntn** [comment]
okay this should be sorted now -- turns out edit.js was alright, but the entityList.js hadn't been updated.  this also fixes a typo in the examples script.

**Comment by hifi-gustavo** [comment]
Build console output for commit: aafed4e1b8470889ac9284bb52547aef83bfeb20 available at: https://gist.github.com/21e8481a17ff1804de42bbd363b48ced<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7785.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7785.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: e68909ce3e65288c27790403693d23a1a6180e5a available at: https://gist.github.com/4555e2a864c2a12d7172fb396e235a65<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7785.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7785.dmg

**Comment by Nex-Pro** [comment]
@ZappoMan  The installer failed integrity on two downloads.

**Comment by ctrlaltdavid** [comment]
Test this please

**Comment by ctrlaltdavid** [comment]
Installer failed for me also; I've kicked off a new build.

**Comment by hifi-gustavo** [comment]
Build console output for commit: e68909ce3e65288c27790403693d23a1a6180e5a available at: https://gist.github.com/21e8481a17ff1804de42bbd363b48ced<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7785.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7785.dmg

**Comment by Nex-Pro** [comment]
##### Platform Win10 desktop 16gb GTXx770 2gb

Works as per the test requirements.  
#### PASS

*Would be nice to be also able to click on the mini icons at the right side*

**Comment by imgntn** [comment]
@Nex-Pro yeah they would be nice toggles

**Pull request merged**